README.md

```diff
@@ -5,20 +5,42 @@
 The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
 
 ### On macOS and Linux
+
 ```bash
 $ source setup.sh
 ```
-### On Windows (Using Git Bash)
+
+### On Windows
+
+Git Bash:
+
 ```bash
 $ source setup.sh --windows
 ```
 
+PowerShell
+
+```bash
+$ ./setup --windows
+```
+
+### Environment variables
+
+To run effectively, update the `.env` file with your application details:
+
+1. `API_KEY` - Used to call the [Trello](https://developer.atlassian.com/cloud/trello/rest/api-group-actions/) API
+1. `API_TOKEN` - Used to call the [Trello](https://developer.atlassian.com/cloud/trello/rest/api-group-actions/) API
+1. `BOARD_ID` - An ID reference to the [board](https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-get) on Trello
+
```

Comment:
Good job updating the README. Is it also worth mentioning that your Trello board needs a list called Done to get the special functionality of that status?

Reply:
Added in [7e28c8a](https://github.com/rysharprules/DevOps-Course-Starter/pull/2/commits/7e28c8a1767f76bb8d9b6ca560c7d6ce52e6b78e)